bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -162,6 +168,39 @@ impl Room {
 
         Ok(())
     }
+
+    pub fn send_reply(
+        &self,
+        msg: String,
+        in_reply_to_event_id: String,
+        txn_id: Option<String>,
+    ) -> Result<()> {
+        let room = match &self.room {
+            MatrixRoom::Joined(j) => j.clone(),
+            _ => bail!("Can't send to a room that isn't in joined state"),
+        };
+
+        let event_id =
+            <&EventId>::try_from(in_reply_to_event_id.as_str()).expect("Failed to create EventId.");
+
+        RUNTIME.block_on(async move {
+            let timeline_event = room.event(event_id).await.expect("Couldn't find event.");
+
+            let event_content = timeline_event
+                .event
+                .deserialize_as::<RoomMessageEvent>()
+                .expect("Couldn't deserialise event");
+
+            let original_message =
+                event_content.as_original().expect("Couldn't retrieve original message.");
```

Comment:
This should also either return an error if it fails, or do something else entirely. Otherwise, it will crash on redacted events.